.vscode/settings.json

```diff
@@ -23,6 +23,7 @@
     "build": true,
     "**/*.js": true
   },
+  "typescript.tsserver.maxTsServerMemory": 8000,
```

Comment:
Can you share some re-producible steps for the issues you had with VSCode, please? I can try to take a deeper look.

Reply:
Sure thing:
1. Pull this branch down and open it in VS Code
2. Get rid of the `typescript.tsserver.maxTsServerMemory` setting fix
3. Close all tabs in and close VS Code
4. Open Quilt in VS Code
5. Open `sewing-kit.config.ts` in the workspace root
6. On my machine at least, the builtin TS/JS plugin hangs and never stops initializing, as noted in the bottom toolbar. The plugin eventually restarts itself a couple times and then crashes, leaving a warning popup

<img width="261" alt="Screen Shot 2020-07-22 at 2 52 57 PM" src="https://user-images.githubusercontent.com/46533681/88216422-14a15980-cc2b-11ea-993c-1496842cf291.png">

In VS Code you can use the command `>Open TS Server log` to see the plugin logs.

Reply:
I was able to repro this on first attempt. I tried a few hacks to see what would happen (eg. move the root `tsconfig.json` to `packages/tsconfig.json`). That seemed to help. I reverted my changes and cleaned out my git env and I'm not able to reproduce this anymore :/

Reply:
It might have something to do with my own local setup then 🤔 @lemonmade @TheMallen is this an issue for you guys as well?

Reply:
I'll try it out after our 1:1